src/epub/text/chapter-4-2.xhtml

```diff
@@ -71,7 +71,7 @@
 			<p>“Don’t go, Pyotr Petrovitch,” said Dounia, “you intended to spend the evening. Besides, you wrote yourself that you wanted to have an explanation with mother.”</p>
 			<p>“Precisely so, Avdotya Romanovna,” Pyotr Petrovitch answered impressively, sitting down again, but still holding his hat. “I certainly desired an explanation with you and your honoured mother upon a very important point indeed. But as your brother cannot speak openly in my presence of some proposals of <abbr epub:type="z3998:name-title">Mr.</abbr> Svidrigaïlov, I, too, do not desire and am not able to speak openly⁠ ⁠… in the presence of others⁠ ⁠… of certain matters of the greatest gravity. Moreover, my most weighty and urgent request has been disregarded.⁠ ⁠…”</p>
 			<p>Assuming an aggrieved air, Luzhin relapsed into dignified silence.</p>
-			<p>“Your request that my brother should not be present at our meeting was disregarded solely at my insistence,” said Dounia. “You wrote that you had been insulted by my brother; I think that this must be explained at once, and you must be reconciled. And if Rodya really has insulted you, then he <em>should</em> and <em>will</em> apologise.”</p>
+			<p>“Your request that my brother should not be present at our meeting was disregarded solely at my instance,” said Dounia. “You wrote that you had been insulted by my brother; I think that this must be explained at once, and you must be reconciled. And if Rodya really has insulted you, then he <em>should</em> and <em>will</em> apologise.”</p>
```

Comment:
This can't be correct, I think this was a printer's typo in the print edition. `instance` is not grammatical here.